internal/backends/mcu/renderer.rs

```diff
@@ -290,24 +315,30 @@ fn prepare_scene(
     runtime_window: Rc<i_slint_core::window::Window>,
     size: PhysicalSize,
     devices: &dyn Devices,
+    cache: &mut PartialRenderingCache,
 ) -> Scene {
     let prepare_scene_profiler = profiler::Timer::new(devices);
-    let mut prepare_scene = PrepareScene::new(
-        size,
-        ScaleFactor::new(runtime_window.scale_factor()),
-        runtime_window.default_font_properties(),
-    );
+    let factor = ScaleFactor::new(runtime_window.scale_factor());
+    let prepare_scene = PrepareScene::new(size, factor, runtime_window.default_font_properties());
+    let mut renderer = i_slint_core::item_rendering::PartialRenderer::new(cache, prepare_scene);
+
+    let mut dirty_region = i_slint_core::item_rendering::DirtyRegion::default();
```

Comment:
Is this variable needed? It seems to me that `renderer.dirty_region` is the true source that's updated in the `compute_dirty_regions()` calls below and the `union` calls appear to make this variable just "tail" it.

Perhaps `into_inner` can "spit" out the "old" renderer and the final computed dirty region?

Reply:
but in case there are popups we need to make the union of all dirty regions.